<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKSerialPortTransmissionItemWidget</class>
 <widget class="QWidget" name="SAKSerialPortTransmissionItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="enableCheckBox">
     <property name="text">
      <string>启用</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="baudRateComboBox"/>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="dataBitscomboBox"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QComboBox" name="stopBitscomboBox"/>
   </item>
   <item row="1" column="9">
    <widget class="QComboBox" name="parityComboBox"/>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="customBaudrateCheckBox">
     <property name="text">
      <string>自定义波特率</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="handleReceiveDataCheckBox">
     <property name="text">
      <string>处理接收到的数据</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>串口设备</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="7">
    <widget class="QComboBox" name="comComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
